addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -2181,6 +2181,60 @@ QUnit.test('chat window: chat header should not be clickable when thread has mul
     );
 });
 
+QUnit.test('chat window should open when receiving a new DM', async function (assert) {
+    assert.expect(4);
+
+    this.data['mail.channel'].records.push({
+        channel_type: 'chat',
+        id:11,
+        is_pinned: false,
+        members: [this.data.currentPartnerId, 11],
+        uuid: 'channel11uuid',
+    });
+    this.data['res.partner'].records.push({
+        id: 11,
+        name: "lol",
+    });
+    this.data['res.users'].records.push({
+        id: 11,
+        partner_id: 11,
+    });
+    await this.start();
+
+    assert.containsNone(
+        document.body,
+        '.o_ChatWindow',
+        "no chat window should initially be open"
+    );
+
+    // simulate receiving the first message on channel 11
+    await afterNextRender(async () => this.env.services.rpc({
+        route: '/mail/chat_post',
+        params: {
+            context: {
+                mockedUserId: 11,
+            },
+            message_content: "new message",
+            uuid: 'channel11uuid',
+        },
+    }));
+    assert.containsOnce(
+        document.body,
+        '.o_ChatWindow',
+        "a chat window should be open now that current user received a new message"
+    );
+    assert.containsOnce(
+        document.body,
+        '.o_Message',
+        "a message should be listed in the just opened chat window"
+    );
+    assert.strictEqual(
+        document.querySelector('.o_Message_content').textContent.trim(),
+        "new message",
+        "the listed message should be the one just received"
+    );
```

Comment:
would remove these assertions, these are out-of-scope of test (which is just about having chat window of DM being open on receiving new message in it, no more no less)